resources/migrations/001_update_migrations.yaml

```diff
@@ -7386,7 +7386,78 @@ databaseChangeLog:
         - dbms:
             type: mysql,mariadb
 
-  # >>>>>>>>>> DO NOT ADD NEW MIGRATIONS BELOW THIS LINE! ADD THEM ABOVE <<<<<<<<<<
+  - changeSet:
+      id: v51.2024-05-22T02:25:57
```

Comment:
This needs to be in v50 if I'm going to find it useful for https://github.com/metabase/metabase/pull/42994